src/lib/hooks/useAvatar.js

```diff
@@ -51,18 +52,16 @@ const useAvatar = avatar => {
 }
 
 export const useUploadedAvatar = () => {
-  const [uploadedAvatar, setUploadedAvatar] = useStoreProp('uploadedAvatar')
+  const { uploadedAvatar } = useAccount()
+  const { updateUploadedAvatar } = useContext(UserContext)
```

Comment:
again we don't need setters for specific vars. please process my previous comments!